### ISSUE TITLE
New SDK

### ISSUE THREAD

**Issue by willqyu** [daveake/pico-tracker#1]
Hi, 

In trying to build your repo, I found that many functions have been changed in newer versions of the SDK.
 
I changed the following to match the new SDK:
- gpio_funcsel() -> gpio_set_function()
- uart_readable -> uart_is_readable()
- adc_enable_temp_sensor() -> adc_set_temp_sensor_enabled()
- adc_input_select() -> adc_select_input()
- gpio_dir() -> gpio_set_dir()

Additionally, GPIO_FUNC_PROC was removed and instead pins initialised with gpio_init(PIN)

Not sure if you're already adapting the code, but I can submit a pull request with your permissions since I've rewritten it to build correctly.

**Comment by daveake** [comment]
Yes, please do that.  The code was written with a pre-release SDK and, add
I'm busy with other projects, I want planning on working on this project
anytime soon.

Dave

**Comment by willqyu** [comment]
Great, I've submitted the pull request:
https://github.com/daveake/pico-tracker/pull/2

**Comment by daveake** [comment]
Great, I'll accept that tomorrow morning when I'm back at my pc.

Dave

**Comment by systematak** [comment]
I would appreciate it if you could combine the codes on a single page and add a practical picture or video of how to send the code to the Ardiuno program.

**Comment by daveake** [comment]
This repo is obsolete as it was based on beta libraries.

It’s for the pi pico not arduino.

It’s modular for a reason; combining to a monolithic single source file
would be opposite to good practice.

**Comment by systematak** [comment]
Well, can you give me some information on how to install pico?

**Comment by systematak** [comment]
www.raspberrypi.com/news/raspberry-pi-pico-balloon-tracker
I collected the electronic parts in the link given, but I couldn't figure out how to transfer the files to the card.
